.gitlab/pipelines/stages/deploy/wait_kubecf.sh

```diff
@@ -7,26 +7,28 @@ source "$(bazel info workspace)/.gitlab/pipelines/runtime/config.sh"
 # shellcheck disable=SC1090
 source "$(bazel info workspace)/.gitlab/pipelines/runtime/binaries.sh"
 
-get_endpoint() {
-  "${KUBECTL}" get endpoints kubecf-router-public \
-    --namespace "${KUBECF_NAMESPACE}" \
-    --output jsonpath='{.subsets[0].addresses[0].ip}' 2> /dev/null
+wait_kubecf() {
+  timeout 1800 bash -c "
+    pod_count() {
+      '${KUBECTL}' get pods --namespace ${KUBECF_NAMESPACE} --no-headers \
+        | wc -l
+    }
+    ready_pod_count() {
+      '${KUBECTL}' get pods --namespace ${KUBECF_NAMESPACE} --no-headers \
+        | awk '{ printf \"%s-%s\n\", \$2, \$3 }' | grep -x '\(.*\)\/\1-Running' \
+        | wc -l
+    }
+    while true; do
+      if [[ \"\$(pod_count)\" != \"0\" ]] && [[ \"\$(pod_count)\" == \"\$(ready_pod_count)\" ]]; then
+        exit 0
+      fi
+      sleep 1
```

Comment:
Do we want to print out some sort of message?
```bash
printf "\r%d of %d pods ready." "$(ready_pod_count)" "$(pod_count)"
```
(Okay probably cache them in variables so that we don't run those things twice every loop)